src/System.Collections.Immutable/src/System/Collections/Immutable/ImmutableArray`1.cs

```diff
@@ -839,11 +850,11 @@ public ImmutableArray<T> RemoveRange(ImmutableArray<T> items)
         public ImmutableArray<T> RemoveRange(ImmutableArray<T> items, IEqualityComparer<T> equalityComparer)
         {
             var self = this;
+            self.ThrowNullRefIfNotInitialized();
```

Comment:
@stephentoub This was the only case in the file where the init check was not at the top of the method, so I thought it may have been a mistake on the part of the original writer.